src/Compilers/CSharp/Portable/Symbols/Source/LocalFunctionSymbol.cs

```diff
@@ -470,24 +470,24 @@ public override ImmutableArray<TypeParameterConstraintClause> GetTypeParameterCo
                     TypeParameters,
                     syntax.TypeParameterList,
                     syntax.ConstraintClauses,
-                    canIgnoreNullableContext,
                     diagnostics);
+
                 lock (_declarationDiagnostics)
                 {
-                    canIgnoreNullableContext = constraints.IgnoresNullableContext();
```

Comment:
>canIgnoreNullableContext = constraints.IgnoresNullableContext(); [](start = 20, length = 64)

We're ignoring the actual `UsedLightWeightTypeConstraintBinding` value from `MakeTypeParameterConstraints()`. Aren't there cases where we call `canUseLightWeightTypeConstraintBinding = true` but `MakeTypeParameterConstraints()` produced a fully bound value (in particular, `null`)?